websites/api-docs-smoke-test/src/content/large-desktop-layout.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Large Desktop Layout
+description: Test components that can flow into two content columns
+wideLayout: true
```

Comment:
This shouldn't be needed, since the website has already the `allowWideContentLayout` option enabled.

Reply:
It's a test case to have the "true" option at least parsed and passed down once - if it's not enabled site-wide you can override it per page. A better test case would be to have one page in the `docs-smoke-test` use the frontmatter with "true" though.